yt_dlp/extractor/rai.py

```diff
@@ -707,7 +719,8 @@ def _real_extract(self, url):
 
 
 class RaiNewsIE(RaiIE):
-    _VALID_URL = rf'https?://(www\.)?rainews\.it/[^?#]+-(?P<id>{RaiBaseIE._UUID_RE})(?:-[^/?#]+)?\.html'
+    _VALID_URL = rf'https?://(www\.)?rainews\.it/(?!articoli)[^?#]+-(?P<id>{RaiBaseIE._UUID_RE})(?:-[^/?#]+)?\.html'
+    _EMBED_REGEX = [r'<iframe[^>]+data-src="(?P<url>/iframe/[^?#]+?\.html)']
```

Comment:
this'll have to test too many iframes. Add the domain to the regex

Reply:
the html code for the iframe is this:
`<iframe class="lazyload" allowfullscreen="true" webkitallowfullscreen="true" mozallowfullscreen="true" allow="encrypted-media *" frameBorder="0" scrolling="no" data-src="/iframe/video/2022/07/euro2022-europei-calcio-femminile-islanda-francia-gol-video-db4bee18-de2c-4ec7-b30a-83c302a43085.html"></iframe>`

as you can see, there is no domain so... maybe I can improve the regex to make it more specific in this way:
`_EMBED_REGEX = [r'<iframe[^>]+data-src="(?P<url>/iframe/[^?#]+?{RaiBaseIE._UUID_RE}\.html)']`

Reply:
ok, done.

It should be more specific.